### PR TITLE
Tayyab

### PR DESCRIPTION
**Pull request by tayyab-ash**
commented a line and added a h3

### CODE REVIEW

File: src/Components/Structure/WelcomeSection.jsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ const WelcomeSection = () => {
         <div className="bg-green-300 ">
           <p className="flex flex-row justify-start">Welcome to</p>
           <h2 className="flex flex-row justify-start text-3xl">Clock Studio</h2>
+          <h3>hehehe</h3>
```
Comment (mtalha7274) [inline src/Components/Structure/WelcomeSection.jsx]: remove this tag

File: src/Components/Structure/WelcomeSection.jsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import "../Design/WelcomeSection.css";
-import welcome from "../Assets/welcome.svg";
+// import welcome from "../Assets/welcome.svg";
```
Comment (mtalha7274) [inline src/Components/Structure/WelcomeSection.jsx]: uncomment this line

**Review by mtalha7274 — CHANGES_REQUESTED** [review]

File: src/Components/Structure/WelcomeSection.jsx (hunk 2 above)
Comment (tayyab-ash) [reply src/Components/Structure/WelcomeSection.jsx]: resolved changes

File: src/Components/Structure/WelcomeSection.jsx (hunk 1 above)
Comment (tayyab-ash) [reply src/Components/Structure/WelcomeSection.jsx]: krdia

**Pull request merged**